### DISCUSSION TITLE
Search bar for Rewards menu

### DISCUSSION THREAD

**Discussion by BubbleVee** [night/betterttv#6242] [Ideas]
On channels with a lot of rewards, it can be hard to scroll through and find the right one for the moment before it's passed. So an option to add a search bar that auto-focuses when the rewards menu is opened to filter down results would be nice to have

![image](https://github.com/night/betterttv/assets/71787375/2d8a6674-f7c0-4139-bea7-ebe01c9827e3) ![image](https://github.com/night/betterttv/assets/71787375/ce5c7059-1ed2-4b2d-878a-f8e17ca32020)

**Comment by jocafamaka** [comment]
This would be great, especially for channels with (many) audio rewards.

My case:
A channel that has reward audios, sometimes something happens that matches audio x, I open the tab and start looking (there are several items), after a while I find the audio, I lost the timing, and now the audio wouldn't make sense.

A search bar would be extremely useful, when you know exactly what you want.
